### PR TITLE
Completion for archlinux-java

### PR DESCRIPTION
**Pull request by JindrichPilar**
Completion script for archlinux-java a tool for selecting default Java runtime (https://wiki.archlinux.org/index.php/java)

### CODE REVIEW

**Comment by syohex** [comment]
Could you add above snippet at end of file and reformat ?

```sh
# Local Variables:
# mode: Shell-Script
# sh-indentation: 2
# indent-tabs-mode: nil
# sh-basic-offset: 2
# End:
# vim: ft=zsh sw=2 ts=2 et
```

**Comment by JindrichPilar** [comment]
Added and reformated

File: src/_archlinux-java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  (set)
+    local java_versions=("${(@f)$(archlinux-java status | tail -n +2 | tr -s ' ' | cut -d ' ' -f2)}")
+    _describe -t output 'Downloads to delete' java_versions
+    ;;
+esac
+
+}
+
+_archlinux-java() {
+local -a commands
```
Comment (syohex) [inline src/_archlinux-java]: Code in function is not indented.

File: src/_archlinux-java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+# Authors
+# -------
+#
+#  * Jindřich Pilař (https://github.com/JindrichPilar)
+#
+# ------------------------------------------------------------------------------
+
+
+_archlinux-java_command_arguments() {
+case $words[1] in
```
Comment (syohex) [inline src/_archlinux-java]: Code in function is not indented.

Comment (syohex) [inline src/_archlinux-java]: That snippet is a part of template file of completion file. I don't know well about vim's indentation behaviour.

Comment (syohex) [inline src/_archlinux-java]: Could you set indentation like other completion files ?

**Pull request merged**